HRIS.Services.Tests/Helpers/EmailHelperUnitTests.cs

```diff
@@ -39,14 +39,15 @@ public void CompileMessageTest()
         Assert.Equal("This is a dynamic body. 2027/07/02", message.Body);
     }
 
-    [Fact]
+    [Fact(Skip = "Needs to be fixed/updated")]
```

Comment:
```suggestion
    [Fact]
```
![image](https://github.com/RetroRabbit/RGO-Server/assets/123597519/2d540936-94bf-48a0-997e-ad59e08dfd25)